components/Themed.tsx

```diff
@@ -23,27 +27,35 @@ type ThemeProps = {
   darkColor?: string;
 };
 
-export type TextProps = ThemeProps & DefaultText['props'];
-export type ViewProps = ThemeProps & DefaultView['props'];
-export type ButtonProps = ThemeProps & DefaultButton['props'];
+export type TextProps = ThemeProps & DefaultText["props"];
+export type ViewProps = ThemeProps & DefaultView["props"];
+export type ButtonProps = ThemeProps & DefaultButton["props"];
 
 export function Text(props: TextProps) {
   const { style, lightColor, darkColor, ...otherProps } = props;
-  const color = useThemeColor({ light: lightColor, dark: darkColor }, 'text');
+  const color = useThemeColor({ light: "black", dark: "black" }, "text");
 
   return <DefaultText style={[{ color }, style]} {...otherProps} />;
 }
 
 export function View(props: ViewProps) {
   const { style, lightColor, darkColor, ...otherProps } = props;
-  const backgroundColor = useThemeColor({ light: lightColor, dark: darkColor }, 'background');
+  const backgroundColor = useThemeColor(
+    { light: "white", dark: "white" },
+    "background"
+  );
 
   return <DefaultView style={[{ backgroundColor }, style]} {...otherProps} />;
 }
 
 export function Button(props: ButtonProps) {
   const { style, lightColor, darkColor, ...otherProps } = props;
-  const color = useThemeColor({light: lightColor, dark: darkColor}, 'text');
-
-  return <DefaultButton style={[{ backgroundColor: color }, style]} {...otherProps} />;
+  const color = useThemeColor({ light: lightColor, dark: darkColor }, "text");
+
+  return (
+    <DefaultButton
+      style={[{ backgroundColor: color }, style]}
+      {...otherProps}
```

Comment:
Removes dark mode for now, since it just causes issues. If we want to at the end, we can add it back